lib/components/Modal/Bottom/modal-image.tsx

```diff
@@ -0,0 +1,33 @@
+import clsx from "clsx";
+
+interface ModalBodyProps {
+    className?: string;
+    src?: string;
+    height?: string;
+}
+
+export const ModalImage = ({
+    children,
+    className,
+    src,
+    height = "var(--temp-static-spacing-210)",
+}: React.PropsWithChildren<ModalBodyProps>) => (
+    <>
```

Comment:
```suggestion
```
no need for this fragment :)

Reply:
It's not working without them. There is no syntax error, image is just not rendering)))